### PR TITLE
[17.0][OU-ADD] payment: migrate to 17.0

### PR DESCRIPTION
**Pull request by daiduongnguyen-odoo**

### CODE REVIEW

File: openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+DEL payment.icon: payment.payment_icon_mpesa
+DEL payment.icon: payment.payment_icon_mtn_mobile_money
+DEL payment.icon: payment.payment_icon_paypal
+DEL payment.icon: payment.payment_icon_sadad
+DEL payment.icon: payment.payment_icon_sepa
+# NOTHING TO DO
+
+NEW payment.method: payment.payment_method_7eleven (noupdate)
+NEW payment.method: payment.payment_method_ach_direct_debit (noupdate)
+NEW payment.method: payment.payment_method_acss_debit (noupdate)
```
Comment (ecino) [inline openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt:140]: This record has been removed https://github.com/odoo/odoo/commit/4b067a43593f3e2153be99d60cef69302601b6ee

Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt:140]: Ok nevermind won't affect the migration process

Comment (zamberjo) [reply openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt:140]: Hello, it seems that if it affects, it gives error in https://github.com/OCA/OpenUpgrade/blob/17.0/openupgrade_scripts/scripts/payment/17.0.2.0/noupdate_changes.xml#L43

```
... OpenUpgrade: payment: error in migration script /opt/odoo/auto/addons/openupgrade_scripts/scripts/payment/17.0.2.0/post-migration.py: while parsing /opt/odoo/auto/addons/openupgrade_scripts/scripts/payment/17.0.2.0/noupdate_changes.xml:42, somewhere inside
....
External ID not found in the system: payment.payment_method_acss_debit
...
ValueError: <class 'ValueError'>: "External ID not found in the system: payment.payment_method_acss_debit" while evaluating
```

Comment (remi-filament) [reply openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt:140]: Hi @duong77476-viindoo thank you for initializing these migration scripts.
I reproduce the error above, but you are right that it is not related to upgrade_analysis files but the noupdate_changes.xml need to be updated here : https://github.com/OCA/OpenUpgrade/blob/b2f4cfe3f6f43676d9e4dfdf28ea56bd272a6a3c/openupgrade_scripts/scripts/payment/17.0.2.0/noupdate_changes.xml#L43
to remove ref('payment.payment_method_acss_debit'), which was removed in commit listed by @ecino above, after you created the migration script (commit is from end of July 24)

Comment (daiduongnguyen-odoo) [reply openupgrade_scripts/scripts/payment/17.0.2.0/upgrade_analysis_work.txt:140]: Yes, thank you. But currently me and my company stop doing migration, we don't have enough resource and will come back when we have more employees i guess. You can create another pr from this and ping me to close

**Pull request closed without merging**